src/gui/dialogs/DialogRadioButton.cpp

```diff
@@ -113,7 +113,15 @@ void RadioButton::draw_n_btns(size_t btn_count) const {
 
     Rect16 splits[4]; //fix size, dont want to use template
     Rect16 spaces[3];
-    rect.VerticalSplit(splits, spaces, btn_count, GuiDefaults::ButtonSpacing);
+    size_t ratio[4];
+
+    for (size_t index = 0; index < btn_count; index++) {
+        string_view_utf8 txt = _((*texts)[index]);
+        ratio[index] = txt.computeNumUtf8CharsAndRewind();
```

Comment:
are you sure btn_count never exceeds 4 items? At least a static_assert would be nice

Reply:
assert added